draft-ietf-oauth-jwt-introspection-response.xml

```diff
@@ -214,155 +214,154 @@ token=2YotnFZFEjr1zCsicMWpAA]]></artwork>
 	  </t>
 	  
 	  <t>If required by its policy, the authorization server MUST authenticate the 
-	  caller and check its authorization to use the token introspection 
+	  caller and validate its authorization to use the token introspection
 	  endpoint.</t>
 
 	</section>
 	
     <section anchor="jwt_response" title="JWT Response">
     
-      <t>The introspection endpoint responds with a JWT, setting the
-          "Content-Type" header to "application/jwt". This JWT is a cryptographically 
-          protected representation of the token introspection response as specified in
-          <xref target="RFC7662"/>.</t>
+     <t>The introspection endpoint responds with a JWT, setting the
+     <spanx style="verb">Content-Type</spanx> HTTP header to
+     "application/token-introspection+jwt" and the JWT <spanx style="verb">typ</spanx>
+     ("type") header to "token-introspection+jwt".</t>
+
+     <t>The JWT MUST include the following top-level claims:
+     <list hangIndent="8" style="hanging">
+         <t hangText="iss">MUST be set to the issuer URL of the authorization
+         server.</t>
+         <t hangText="aud">MUST identify the resource server receiving the token
+         introspection response.</t>
+         <t hangText="token_introspection">A JSON object containing the members
+         of the token introspection response, as specified in the "OAuth Token
+         Introspection Response" registry established by <xref target="RFC7662"/>
+         as well as other members. The separation of the introspection members into
+         a dedicated containing JWT claim is intended to prevent conflict and confusion
+         with top-level JWT claims that may bear the same name.
+         <vspace blankLines="1" />
+         If the access token is invalid, expired, revoked, or is not intended for the
+         calling resource server (audience), the authorization server MUST set the value of the
+         <spanx style="verb">active</spanx> member in the <spanx style="verb">token_introspection</spanx>
+         claim to <spanx style="verb">false</spanx>. Otherwise, the <spanx style="verb">active</spanx>
```

Comment:
```suggestion
         claim to <spanx style="verb">false</spanx>. The container MUST not contain any other claims in this case. Otherwise, the <spanx style="verb">active</spanx>
```